wetterdienst/util/cache.py

```diff
@@ -12,14 +12,18 @@
 log = logging.getLogger()
 
 # Python on Windows has no "fcntl", which is required by the dbm backend.
-# TODO: Make backend configurable, e.g. better use Redis.
+
 platform = platform.system()
 backend = "dogpile.cache.dbm"
 if platform == "Windows":
```

Comment:
Change this here to use in-memory caching only
 ```python
 if platform == "Windows" or os.environ['WD_CACHE_DISABLE']:
 ```

Reply:
Done